en/06_Project_Governance/08_Public_API.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Definition of Public API
+summary: A definition of what we consider to be "public API" which is protected by semantic versioning
+icon: code
+---
+
+# Definition of public API
+
+The following is a definition of what we consider to be "public API". Public API is protected by semantic versioning.
+
+New public API will not be introduced in patches, but can be introduced in minor and major releases.
+
+Existing public API will not be removed or altered in patches or minor releases, but can be removed or altered in major releases.
+
+## Explicitly included
+
+These are explicitly included in our definition of public API (unless excluded below):
+
+* **global** functions, constants, and variables
+* namespaces, classes, interfaces, enums, and traits
+* public and protected scope (including methods, properties and constants)
+* private static class properties (considered to be configuration variables)
+* yml configuration file structure and value types
+* extension hooks (e.g. `$this->extend('someExtensionHook'));`)
+
+## Explicitly excluded
+
+These are explicitly _not_ public API:
+
+* private scope (with the exception for `private static` properties which aren't annotated with `@internal`)
+* all entities marked as `@internal`
+* yml configuration file default values
```

Comment:
```suggestion
* yml configuration file default values
```
There's no difference between yml config and private static config - this reads like yml config is out of scope though private static class properties are in scope (it's listed above).  Need to clarify

Reply:
Yeah I was just copying what was already there and moving it to this page, but if you're saying updating this is in scope then I'll update it.

Done.